### ISSUE TITLE
Wiped my whole steamapps folder when cancelling Mod Organizer 2 install

### ISSUE THREAD

**Issue by onlyreportingissues** [flathub/net.lutris.Lutris#301]
Wanted to choose another folder. When cancelling the install, it asked me if I wanna delete the install files & I clicked yes.

Lutris went on a rampage then & deleted around 500gb of files.

**Comment by strycore** [comment]
Yes, the current way the file deletion is handled for installation cancellation is unsafe, it doesn't use the same deletion code we use for game uninstalls, which would be better but still not ideal.
We're going to move to a file list based deletion where we keep track of the files added to a folder and only delete those in both installation cancellation and game removal.

**Comment by onlyreportingissues** [comment]
Ok, sounds promising. Thanks.
